Tried to pull image manifest for registry-1.docker.io/arm64v8/openjdk:8 but failed because: manifest unknown

<!--
Before filing an issue, please reproduce with the latest version of Jib.

If you are encountering errors pulling or pushing to remote registries,
please check our Frequently Asked Questions before filing an issue:

    https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#registry-errors
-->

**Environment**:

- *Jib version:1.7.0*
- *Build tool:apache-maven-3.6.1* 
- *OS: arm linux*
```  
OS name: "linux", version: "4.9.140-tegra", arch: "aarch64", family: "unix"
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.2 LTS
Release:        18.04
Codename:       bionic
```

**Description of the issue**:
 Failed to execute goal com.google.cloud.tools:jib-maven-plugin:1.7.0:build (default) on project xxx: Tried to pull image manifest for registry-1.docker.io/arm64v8/openjdk:8 but failed because: manifest unknown

**`jib-maven-plugin` Configuration**: <!-- Delete this section if not used -->
```xml
 <plugin>
                <groupId>com.google.cloud.tools</groupId>
                <artifactId>jib-maven-plugin</artifactId>
                <version>1.7.0</version>
                <configuration>
                    <from>
                        <image>arm64v8/openjdk:8</image>
                        <!-- <image>openjdk:alpine</image> -->
                    </from>
                    <container>
                        <mainClass>com.xxx.xxx.xxx</mainClass>
                        <jvmFlags>
                            <jvmFlag>-server</jvmFlag>
                        </jvmFlags>
                        <!--使用该参数将镜像的创建时间与系统时间对其-->
                        <useCurrentTimestamp>true</useCurrentTimestamp>
                    </container>
                    <!--如果私有镜像仓库没有启用https，设置allowInsecureRegistries参数为true-->
                    <allowInsecureRegistries>false</allowInsecureRegistries>
                    <to>
                        <image>docker.io/xxx/${project.artifactId}</image>
                        <tags>
                            <tag>${project.version}</tag>
                        </tags>
                        <auth>
                            <username>xxx</username>
                            <password>xxx</password>
                        </auth>
                    </to>
                </configuration>
                <executions>
                    <execution>
                        <phase>package</phase>
                        <goals>
                            <goal>build</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>

```

**Log output**: <!-- If applicable, provide relevant log output -->
```
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  52.376 s
[INFO] Finished at: 2019-11-13T16:18:54+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.cloud.tools:jib-maven-plugin:1.7.0:build (default) on project xxx: Tried to pull image manifest for registry-1.docker.io/arm64v8/openjdk:8 but failed because: manifest unknown | If this is a bug, please file an issue at https://github.com/GoogleContainerTools/jib/issues/new: 404 Not Found
[ERROR] {"errors":[{"code":"MANIFEST_UNKNOWN","message":"manifest unknown"}]}
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :xxx
```

--- a/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java
+++ b/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java
@@ -273,7 +273,7 @@ class PullBaseImageStep implements Callable<ImageAndRegistryClient> {
           buildContext.getBaseImageConfiguration().getImage()
               + " is a manifest list, but the list does not contain an image manifest for amd64/linux."
               + " If your intention was to use a non-amd64/linux base image,"
-              + " see  https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image"
+              + " see https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image"
               + " to learn how to specify a manifest instead of a manifest list, until Jib fixes"
               + " https://github.com/GoogleContainerTools/jib/issues/1567 to allow specifying architecture and OS.";
       buildContext.getEventHandlers().dispatch(LogEvent.error(errorMessage));

--- a/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java
+++ b/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java
@@ -270,8 +270,12 @@ class PullBaseImageStep implements Callable<ImageAndRegistryClient> {
     List<String> digests = manifestListTemplate.getDigestsForPlatform("amd64", "linux");
     if (digests.size() == 0) {
       String errorMessage =
-          "Unable to find amd64/linux manifest in manifest list at: "
-              + buildContext.getBaseImageConfiguration().getImage();
+          buildContext.getBaseImageConfiguration().getImage()
+              + " is a manifest list, but the list does not contain an image manifest for amd64/linux."
+              + " If your intention was to use a non-amd64/linux base image,"
+              + " see  https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-do-i-specify-a-platform-in-the-manifest-list-or-oci-index-of-a-base-image"
+              + " to learn how to specify a manifest instead of a manifest list, until Jib fixes"
+              + " https://github.com/GoogleContainerTools/jib/issues/1567 to allow specifying architecture and OS.";
       buildContext.getEventHandlers().dispatch(LogEvent.error(errorMessage));
       throw new RegistryException(errorMessage);
     }

